packages/relic/test/headers/new/special/accept_encoding_header_test.dart

```diff
@@ -0,0 +1,262 @@
+import 'dart:io';
+import 'package:test/test.dart';
+import 'package:relic/src/headers/headers.dart';
+import 'package:relic/src/relic_server.dart';
+import '../../headers_test_utils.dart';
+
+void main() {
+  group('Given an Accept-Encoding header with the strict flag true', () {
+    late RelicServer server;
+
+    setUp(() async {
+      try {
+        server = await RelicServer.bind(InternetAddress.loopbackIPv6, 0);
+      } on SocketException catch (_) {
+        server = await RelicServer.bind(InternetAddress.loopbackIPv4, 0);
+      }
+    });
+
+    tearDown(() => server.close());
+
+    test(
```

Comment:
Question: Is it enough with a single test here that reads `when invalid Accept-Encoding header is passed...`? Since we already validate what "invalid" means in the other tests.

The bad quality test should covered in the other tests right?